### PR TITLE
🔒 feat: add option to disable TLS for LDAP authentication

### LINKED ISSUE #3243: Enhancement: Allow disabling LDAP tls verification
### What features would you like to see added?

If someone doesn't have the LDAP server certificates and would still want to support LDAP authentication. It would be useful to allow disabling tls verification for LDAP authentication strategy. 

### More details

A option to disable LDAP TLS verification can be helpful in some cases.
```
tlsOptions: {
      rejectUnauthorized: LDAP_REJECT_UNAUTHORIZED, // Disables TLS verification (not recommended)
   }
```

### Which components are impacted by your request?

Plugins

### Pictures

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by ravi-katiyar**
Closes: https://github.com/danny-avila/LibreChat/issues/3243

## Summary

This PR allows to disable tls when LDAP authentication is used.

## Change Type

- [x] New feature
- [x] This change requires a documentation update

## Checklist

- [x] My code adheres to this project's style guidelines
- [x] I have performed a self-review of my own code
- [x] I have made pertinent documentation changes
- [x] My changes do not introduce new warnings
- [x] Local unit tests pass with my changes
- [x] [A pull request for updating the documentation has been submitted](https://github.com/LibreChat-AI/librechat.ai/pull/89).

### CODE REVIEW

File: api/strategies/ldapStrategy.js [hunk 1]
```diff
@@ -41,6 +42,7 @@ if (LDAP_ID) {
 if (LDAP_USERNAME) {
   searchAttributes.push(LDAP_USERNAME);
 }
+const rejectUnauthorized = LDAP_TLS_REJECT_UNAUTHORIZED !== 'false';
```
Comment (berry-13) [inline api/strategies/ldapStrategy.js]: can you please use
```
const { isEnabled } = require('~/server/utils');
```

**Review by berry-13 — CHANGES_REQUESTED** [review]

File: api/strategies/ldapStrategy.js (hunk 1 above)
Comment (ravi-katiyar) [reply api/strategies/ldapStrategy.js]: Does it look ok now ?

Comment (berry-13) [reply api/strategies/ldapStrategy.js]: yes, thanks

**Review by berry-13 — APPROVED** [review]

**Comment by ravi-katiyar** [comment]
@danny-avila please let me know if anything is pending at my end for this.

**Comment by ravi-katiyar** [comment]
@berry-13 please let me if there's anything pending on this , I would like to get this merged.

**Comment by danny-avila** [comment]
@ravi-katiyar can you add relevent documentation to

https://www.librechat.ai/docs/configuration/authentication/ldap ?

https://github.com/LibreChat-AI/librechat.ai/blob/main/pages/docs/configuration/authentication/ldap.mdx ?


Since you area also adding an .env variable, you should update this too: https://github.com/LibreChat-AI/librechat.ai/blob/main/pages/docs/configuration/dotenv.mdx

**Comment by ravi-katiyar** [comment]
> @ravi-katiyar can you add relevent documentation to
> 
> https://www.librechat.ai/docs/configuration/authentication/ldap ?
> 
> https://github.com/LibreChat-AI/librechat.ai/blob/main/pages/docs/configuration/authentication/ldap.mdx ?
> 
> Since you area also adding an .env variable, you should update this too: https://github.com/LibreChat-AI/librechat.ai/blob/main/pages/docs/configuration/dotenv.mdx

@danny-avila , Thanks for pointing that out. PR submitted to update the docs https://github.com/LibreChat-AI/librechat.ai/pull/89

**Review by danny-avila — APPROVED** [review]

**Pull request merged**